docs/source/python/development.rst

```diff
@@ -267,6 +267,26 @@ we recommend the following:
 #. Add ``-DARROW_ORC=on`` to the CMake flags.
 #. Add ``--with-orc`` to the ``setup.py`` flags.
 
+
+Building documentation
+----------------------
+
+To build the documentation:
+
+#. build the doxigen cpp api
+
+   .. code-block:: shell
+
+      cd arrow/cpp/apidoc
```

Comment:
Just "cd cpp/apidoc". Top-level "arrow" is not part of the source tree.

Reply:
done, but I have seen in other place like:
cd arrow/python
python setup.py build_ext --build-type=$ARROW_BUILD_TYPE \
       --with-parquet --with-plasma --inplace